assets/blocks/email-editor.test.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Internal dependencies
+ */
+import {} from './email-editor';
```

Comment:
```suggestion
import './email-editor';
```
or you can use the exposed function.